packages/twilio-run/src/utils/deployInfoCache.ts

```diff
@@ -71,9 +72,15 @@ export function updateDeployInfoCache(
     deployInfoCacheFileName
   );
 
+  if (currentDeployInfoCache.hasOwnProperty(username) && region === 'us1') {
+    debug('Invalid format for deploy info key. Overriding with region us1');
```

Comment:
i'm confused about this block, is the username always an account sid? if so, it seems like this would only be invalid if just the account sid key is in the cache, but the region is not `us1`? lmk if i'm misunderstanding though.